BitwardenShared/UI/Auth/Login/LoginWithDevice/LoginWithDeviceProcessor.swift

```diff
@@ -193,6 +193,8 @@ final class LoginWithDeviceProcessor: StateProcessor<
                 coordinator.navigate(to: .twoFactor(state.email, unlockMethod, authMethodsData, nil))
             case .twoFactorProvidersNotConfigured:
                 services.errorReporter.log(error: error)
+            case .newDeviceNotVerified:
+                services.errorReporter.log(error: error)
```

Comment:
🤔 I guess the TODO applies here as well, right? `TODO: PM-8222`?
Also could tests be added that the error reporter is logging the error here?

Reply:
Well this will never happen, the device error is only sent by the server if the user is logging in using master password. The other auth flows will not trigger it. But I guess I can add the test to the error reporting.